common/cards/alter-egos-iii/hermits/poepoeskizz-rare.ts

```diff
@@ -65,7 +65,7 @@ class PoePoeSkizzRare extends Card {
 					})
 					jumpscareAttack.addDamage(component.entity, 20)
 					jumpscareAttack.shouldIgnoreCards.push(query.card.entity(component.entity))
-					executeExtraAttacks(game, [jumpscareAttack])
+					attack.addNewAttack(jumpscareAttack)
```

Comment:
The original form is correct because `attack` already finished executing after the pick request was fulfilled